test/integration/functional_test.go

```diff
@@ -2067,7 +2071,7 @@ func startMinikubeWithProxy(ctx context.Context, t *testing.T, profile string, p
 		memoryFlag = "--memory=6000"
 	}
 	// passing --api-server-port so later verify it didn't change in soft start.
-	startArgs := append([]string{"start", "-p", profile, memoryFlag, fmt.Sprintf("--apiserver-port=%d", apiPortTest), "--wait=all"}, StartArgs()...)
+	startArgs := append([]string{"start", "-p", profile, memoryFlag, fmt.Sprintf("--apiserver-port=%d", apiPortTest), "--wait=all", "--disable-optimizations=true"}, StartArgs()...)
```

Comment:
I suggest not using functional test for testing "disable-optimizations" because we rely on this test to be stable for "Default Use Case of most users", feel free to combine it with any other test that is not a main one

Reply:
Moved to integration